Mathlib/Algebra/Group/Pointwise/Set.lean

```diff
@@ -777,6 +777,10 @@ lemma smul_set_iUnion₂ (a : α) (s : ∀ i, κ i → Set β) :
 lemma smul_set_iInter_subset (a : α) (t : ι → Set β) : a • ⋂ i, t i ⊆ ⋂ i, a • t i :=
   image_iInter_subset _ _
 
+@[to_additive]
+lemma smul_set_sInter_subset (a : α) (S : Set (Set β)) :
+    a • ⋂₀ S ⊆ ⋂ s ∈ S, (fun x => a • x) '' s := image_sInter_subset S (fun x => a • x)
```

Comment:
Can you also prove
```
lemma smul_sInter_subset (s : α) (T : Set (Set β)) :
    as • ⋂₀ T ⊆ ⋂ t ∈ T, s • t := sorry
```

Reply:
@YaelDillies how is that different to `smul_set_sInter_subset`? What is `as`?

Reply:
Sorry, corrected

Reply:
Done.